samples/bookinfo/src/details/Gemfile.lock

```diff
@@ -0,0 +1,14 @@
+GEM
+  remote: https://rubygems.org/
+  specs:
+    webrick (1.8.1)
+
+PLATFORMS
+  ruby
+  x86_64-linux
```

Comment:
excuse my ignorance, I'm not a ruby expert. can/should we include `aarch64-linux` so we can build multi-arch samples?